components/app/BusinessInfo.jsx

```diff
@@ -89,84 +91,117 @@ const BusinessLink = styled.a`
   font-weight: 600;
 `
 
+const amendLinks = (text) => {
+    const regexWeb = /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/g;
+    const regexEmail = /([^@\s]+@[^@\s]+\.[^@\s]+)/gm;
+    const webReplace = reactStringReplace(text, regexWeb, (match, i) => {
+        const pattern = /^((http|https):\/\/)/;
+
+        if(!pattern.test(match)) {
+            const linkMatch = "https://" + match;
+            return  <a href={linkMatch}>{match}</a>
+        }
+       return  <a href={match}>{match}</a>
+    })
+
+     const replacedText = reactStringReplace(webReplace, regexEmail, (match, i) => (
+         <a href={`mailto:${match}`}>{match}</a>
+    ))
+    return replacedText;
+}
+
+
+
 const Tag = ({ type, ...props }) => (
-  <Box bg="rgba(103, 128, 159, 1)" py={1} px={16} borderRadius={4} {...props}>
-    <TagText color="#fff">
-      {type}
-    </TagText>
-  </Box>
+    <Box bg="rgba(103, 128, 159, 1)" py={1} px={16} borderRadius={4} {...props}>
+        <TagText color="#fff">
```

Comment:
Hi. Seems that an editor config for 4 spaces has slipped into this PR. Maybe worth deciding on a project wide `.editorconfig` for tab width, etc. There is a `.prettierrc` config for tab width, I'm not a big fan of prettier though, especially if it isn't being run in a build step or CI. I'll open an issue about it

Reply:
Yes I just realised this when I did the PR - yes it'll be good to have a consistent config!